src/printer.js

```diff
@@ -1033,15 +1054,14 @@ function printNode(path, options, print) {
             ? concat([
                 " {",
                 indent(
-                  concat(
-                    path.map(adaptationPath => {
-                      return concat([
-                        line,
-                        print(adaptationPath),
-                        lineShouldEndWithSemicolon(adaptationPath) ? ";" : ""
-                      ]);
-                    }, "adaptations")
-                  )
+                  concat([
+                    line,
+                    path.call(
+                      adaptationsPath =>
+                        printLines(adaptationsPath, options, print),
+                      "adaptations"
```

Comment:
indentation issue?

Reply:
so this is actually how prettier formats it haha - `"adaptations"` is the second argument for `call()`, the inline arrow function is the 1st argument. I agree its a little muddy though...

Reply:
Oh, right ^_^